OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -60,5 +60,12 @@ public void Resupply()
 		{
 			Self.QueueActivity(new ResupplyAircraft(Self));
 		}
+
+		[ScriptActorPropertyActivity]
+		[Desc("Returns true if the aircraft circling over it's home base.")]
+		public bool IsOnHoldingPattern
```

Comment:
Just call it `IsCircling`. Also the description is misleading (the "over its home base" part).